test-app/lib/Router.coffee

```diff
@@ -0,0 +1,22 @@
+Session.setDefault '___isPhantomjs___', false if Meteor.isClient
+
+Router.onAfterAction ->
+  Meteor.isReadyForSpiderable = true if @ready()
+
+Router.configure
+  notFoundTemplate: '_404'
+
+Router.plugin 'dataNotFound', 
+  notFoundTemplate: Router.options.notFoundTemplate
+
+if Meteor.isClient
+  Router.onBeforeAction -> 
+    if _.has @params.query, '___isRunningPhantomJS___'
+      Session.set '___isPhantomjs___', true
```

Comment:
Agree, you can remove, or create other much smaller app with only those lines and left a link in docs to it